custom/icds_reports/utils/aggregation_helpers/distributed/daily_feeding_forms_child_health.py

```diff
@@ -26,6 +26,14 @@ def aggregation_query(self):
             "state_id": self.state_id,
         }
 
+        # This query has a strange query plan so there's a few things to note.
+        # This is joined on the daily_attendance table.
+        # The daily_attendance aggregation only includes the most recently submitted form for each day.
+        # Often an AWW may submit multiple daily attendance forms in a day,
+        #   so we choose the last form for each AWW's day.
+        # Because the result set of docs is actually coming from daily_attendance,
+        #   the JOIN uses the primary key (supervisor_id, doc_id, repeat_iteration).
+        # Because of this, the UCR does not have an index on (state_id, timeend)
```

Comment:
I always forgot about why this JOIN happens so hopefully this helps out in the future.

@snopoke curious if this explanation makes sense to you as someone who probably hasn't looked at this in the past

Reply:
yea, that makes sense to me